app/src/Topbar.re

```diff
@@ -32,9 +34,12 @@ module Navigation = {
 [@react.component]
 let make = () =>
   Router.(
-    <header className=Styles.topBar>
-      <Link page=Root> <Atom.Icon.Logo /> </Link>
-      <Navigation />
-      <JoinNetwork />
+    <header className=Styles.header>
+      <Container.TwoColumns>
+        ...(
+             <> <Link page=Root> <Atom.Icon.Logo /> </Link> <Navigation /> </>,
+             <JoinNetwork />,
```

Comment:
Is this different from?

``` jsx
<Link page=Root><Atom.Icon.Logo /></Link>
<Navigation />
<JoinNetwork />
```

Reply:
Yes, `Container.TwoColumns` expects a tuple of two elements, one for the left side and another one for the right side. Also reason-react expects children to always be an array, hence the spread operator.

You can find out more about it [here](https://github.com/reasonml/reason-react/blob/master/docs/children.md#children-spread).

Reply:
Ah sorry, missed the that's a tuple, sneaky semis.